### DISCUSSION TITLE
What is the plan for tokenIssuerUrl and tokenIssuerValidationUrl in future releases

### DISCUSSION THREAD

**Discussion by phil-hotchkiss** [mia-platform/community#388] [Q&A] [answered]
### Description

According to your documentation https://docs.mia-platform.eu/docs/runtime_suite/http-proxy-manager/configuration, you say that you will be deprecating the tokenIssuerUrl and tokenIssuerValidationUrl features in future releases. There doesn't seem to be any other properties that would enable you to acquire a client credential token from an authorization server. What are your plans for these values as referenced in the documentation? 

![image](https://github.com/mia-platform/community/assets/136851567/370906a1-6f2c-43bf-8b46-042a11fef55c)

**Comment by fredmaggiowski** [comment, marked as answer]
Hi @phil-hotchkiss, thanks for reaching out!

This deprecation warning is actually outdated (and wrong). We changed the configuration from having `tokenIssuerUrl` and `tokenIssuerValidationPath` to only require the `tokenIssuerValidationUrl`.

With this warning we wanted to notify that `tokenIssuerValidation*Path*` was going to be removed (and it has already been removed with v2).

You can safely ignore the warning (as we are pulling it down from the doc asap). If you have further question don't hesitate to ask.

Sorry for the misleading information!

Cheers!

**Reply by phil-hotchkiss** [reply]
Thanks for the clarification!

**Reply by fredmaggiowski** [reply]
You're welcome!

Doc update: https://github.com/mia-platform/documentation/pull/1125
